configuration/uncontrast_initial_data_types.py

```diff
@@ -36,22 +36,28 @@
     "Discrimination",
     "Detection",
     "Task is irrelevant to the effect",
```

Comment:
do changes here mean we need to remigrate?

Reply:
yes, as this is still a process of finding out what's missing here. Every time we load and reload the data we need to run existing migrations anyway. But we don't have to **make** the migrations. When I finish with the data corrections there will be a final version with no changes.